venia-integration-tests/src/fixtures/productPage/index.js

```diff
@@ -7,3 +7,11 @@ export const productValeriaTwoLayeredTank = {
     name: 'Valeria Two-Layer Tank',
     url: '/valeria-two-layer-tank.html'
 };
+
+/**
+ * Sample simple product
+ */
+export const silverAmorBandleSet = {
```

Comment:
```suggestion
export const silverAmorBangleSet = {
```

Will need to be updated where used as well